Sources/PointFree/About.swift

```diff
@@ -21,7 +21,7 @@ private let aboutView = View<RequestContext<Prelude.Unit>> { ctx in
         ]),
       body(navView.view(ctx) + [
         gridRow([
-          gridColumn(sizes: [.xs: 12], [
+          gridColumn(sizes: [.mobile: 12], [
```

Comment:
went ahead and updated the grid (and all of design systems) to use the new breakpoints. really slimmed down the generated css!